src/mscorlib/src/System/PinnableBufferCacheEventSource.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+namespace System
+{
+    [EventSource(Guid = "38ed3633-5e3f-5989-bf25-f0b1b3318c9b", Name = "Microsoft-DotNETRuntime-PinnableBufferCache-System")]
+    internal sealed class PinnableBufferCacheEventSource : EventSource
```

Comment:
This is also nice-to-have change unrelated to the ArrayPool problem.

Reply:
There is a perf implication here though- do we really need to roll this one back?

Reply:
- Nobody missed this tracing in CoreCLR.
- We do not have a test coverage for it.
- Tracing related changes has been root cause of quite a few issues lately.

Reply:
I can pull the content, but what about the explicit guid construction?

Reply:
Didn't inherit from `EventSource` before so would be no cost? (Presumably would want to keep Guid for 2.2 though?)

Reply:
> Didn't inherit from EventSource before so would be no cost?

Derp, missed that. :)